ensembl_dbcopy/urls.py

```diff
@@ -18,7 +18,7 @@
 app_name = 'ensembl_dbcopy'
 
 urlpatterns = [
-    path('dbcopy/submit/', SubmitView.as_view()),
```

Comment:
@marcoooo: I think this will break the templates: https://github.com/Ensembl/ensembl-production-services/blob/new_copy_service/ensembl_dbcopy/templates/detail.html#L6